### PR TITLE
docs: fix weird helm upgrade example

### PR DESCRIPTION
**Pull request by hiroki-sawano**
If I understand correctly, `--version` for this specific example should be set to `v0.6` according to the instruction.

### CODE REVIEW

**Comment by consideRatio** [comment]
Correct @hiroki-sawano, that documentation was written in a way I didn't understand clearly. This sentence specifically doesn't really make sense:

```
For example, to upgrade to v0.6, enter and substituting `<helm-release-name>` and version v0.6:
```

Could you try rewrite that sentence along with this change? I'm thinking something like...

_For example, to upgrade to version ... with a helm release name of ... in the k8s namespace of ...:_

**Comment by hiroki-sawano** [comment]
@consideRatio 
Thanks for the reply.
> Could you try rewrite that sentence along with this change? 

Sure. Please check the changes I added: https://github.com/jupyterhub/zero-to-jupyterhub-k8s/pull/2331/commits/5a7db2394f6bb678b9e5827699a93536e0a39c2c

File: doc/source/administrator/upgrading.md [hunk 1]
```diff
@@ -66,10 +66,10 @@ To run the upgrade:
 helm upgrade --cleanup-on-fail <helm-release-name> jupyterhub/jupyterhub --version=<chart-version> --values config.yaml --namespace <k8s-namespace>
 ```
 
-For example, to upgrade to v0.6, enter and substituting `<helm-release-name>` and version v0.6:
+For example, to upgrade to version `v0.6` with a helm release name of `jhub` in the k8s namespace of `jhub`:
 
 ```
-helm upgrade --cleanup-on-fail <helm-release-name> jupyterhub/jupyterhub --version=<chart-version> --values config.yaml --namespace <k8s-namespace>
+helm upgrade --cleanup-on-fail jhub jupyterhub/jupyterhub --version=v0.6 --values config.yaml --namespace jhub
```
Comment (consideRatio) [inline doc/source/administrator/upgrading.md:72]:
```suggestion
helm upgrade --cleanup-on-fail jhub jupyterhub/jupyterhub --version=1.1.1 --values config.yaml --namespace jhub
```

File: doc/source/administrator/upgrading.md [hunk 2]
```diff
@@ -66,10 +66,10 @@ To run the upgrade:
 helm upgrade --cleanup-on-fail <helm-release-name> jupyterhub/jupyterhub --version=<chart-version> --values config.yaml --namespace <k8s-namespace>
 ```
 
-For example, to upgrade to v0.6, enter and substituting `<helm-release-name>` and version v0.6:
+For example, to upgrade to version `v0.6` with a helm release name of `jhub` in the k8s namespace of `jhub`:
```
Comment (consideRatio) [inline doc/source/administrator/upgrading.md:69]:
```suggestion
For example, to upgrade to version `1.1.1` with a helm release name of `jhub` in the k8s namespace of `jhub`:
```

**Review by consideRatio — APPROVED** [review]

**Comment by consideRatio** [comment]
Thanks @hiroki-sawano! I went ahead and used version 1.1.1 instead of version v0.6 so there is less confusion if there is supposed to be a `v` prefix or not, which it isn't in the more recent versions of this Helm chart.

**Comment by consideRatio** [comment]
Thank you @hiroki-sawano !!! Welcome to the jupyterhub open source ecosystem!

**Pull request merged**